Makefile

```diff
@@ -1,5 +1,14 @@
 DOCKER_IMAGE ?= lev-report
 DOCKER_DB_IMAGE ?= lev-report-mock-db
+postgres_hostname=localhost
+postgres_port=5432
+postgres_username=lev
+postgres_password=lev
+postgres_database=lev
```

Comment:
I wonder if we can share this config with the docker-compose? I seem to remember something about a `.env` file.